scripts/backup.sh

```diff
@@ -1,10 +1,30 @@
 #!/bin/sh
 
+############ VARIABLE DEFINITIONS ############
+BACKUP_DIR=/home/dev/backups/
+MEDIA_DIR=/home/dev/BP2021RH1/projects/bp-strapi/public/uploads
+DB_NAME=bp-strapi
+DB_USER=postgres
+##############################################
+
+
+############ CODE ############
+# Find the last backup, if none exists, it will just continue to make a full backup
+LAST_BACKUP_FOLDER=$(ls -td -- "$BACKUP_DIR"* | head -n 1)
+
+echo "last backup is placed in folder $LAST_BACKUP_FOLDER"
```

Comment:
I think it is fair that garbage like "no such file found" is echoed, so it clear in the logs that the first full backup is missing.